plugins/catalog-backend/migrations/20220616202842_refresh_keys.js

```diff
@@ -32,13 +32,12 @@ exports.up = async function up(knex) {
       .onDelete('CASCADE')
       .comment('A reference to the entity that the refresh key is tied to');
     table
-      .string('key')
+      .text('key')
       .notNullable()
       .comment(
         'A reference to a key which should be used to trigger a refresh on this entity',
       );
     table.index('entity_id', 'refresh_keys_entity_id_idx');
-    table.index('key', 'refresh_keys_key_idx');
```

Comment:
I've removed the index on the key column from the refresh_keys migration due to its potential size and this error: `BLOB/TEXT column 'key' used in key specification without a key length`. Should we keep the index and specify a key length instead? I'm unsure what length to use.